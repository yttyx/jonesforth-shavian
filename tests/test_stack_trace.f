( -*- 𐑑𐑧𐑒𐑕𐑑 -*- )

: 𐑑𐑧𐑕𐑑4 𐑐𐑮𐑦𐑯𐑑-𐑕𐑑𐑨𐑒-𐑑𐑮𐑱𐑕 ;

: 𐑑𐑧𐑕𐑑3 𐑑𐑧𐑕𐑑4 1 2 + . 𐑒𐑮 𐑑𐑧𐑕𐑑4 ;

: 𐑑𐑧𐑕𐑑2 𐑑𐑧𐑕𐑑3 𐑑𐑧𐑕𐑑3 ;

: 𐑑𐑧𐑕𐑑 𐑑𐑧𐑕𐑑2 ;
