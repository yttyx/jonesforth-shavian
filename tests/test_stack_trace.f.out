𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+0 𐑑𐑧𐑕𐑑2+0 𐑑𐑧𐑕𐑑+0 
3 
𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+32 𐑑𐑧𐑕𐑑2+0 𐑑𐑧𐑕𐑑+0 
𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+0 𐑑𐑧𐑕𐑑2+4 𐑑𐑧𐑕𐑑+0 
3 
𐑑𐑧𐑕𐑑4+0 𐑑𐑧𐑕𐑑3+32 𐑑𐑧𐑕𐑑2+4 𐑑𐑧𐑕𐑑+0 
